http/codegen/openapi/v3/builder_test.go

```diff
@@ -318,12 +318,14 @@ func matchesResponse(t *testing.T, r *ResponseRef, types map[string]*openapi.Sch
 		}
 		matchesHeader(t, h, types, exp)
 	}
-	ct, ok := v.Content["application/json"]
-	if !ok {
-		t.Error("missing response content, expected application/json")
-		return
+	if expected.Type.Type != "" {
```

Comment:
Just double-checking, is this supposed to be `expected.Type.Type` or just `expected.Type`?

Reply:
Looking at the way that test struct was designed, it should be `expected.Type.Type`. We could refactor the tests to take a struct pointer instead of the actual struct so that we can check it like `expected.Type == nil`.

Reply:
Cool, thanks for the explanation.